lib/Walli.ts

```diff
@@ -14,12 +15,13 @@ class Walli {
   private logger: Logger;
   private btcdClient: BtcdClient;
   private config: Config;
+  private grpcServer: GrpcServer;
 
   constructor(config: ConfigType) {
     this.config = new Config().load(config);
     this.logger = new Logger(this.config.logfile, this.config.loglevel);
-    console.log(this.config);
     this.btcdClient = new BtcdClient(this.config.rpc);
+    this.grpcServer = new GrpcServer(this.logger);
```

Comment:
Should I start the gRPC server in the `Walli.start()` function?
Or should I initialize it in an express server like they do in xud?

Reply:
In `Walli.start()`. The express server is for using the gRPC with a HTTP client and we don't need this for now imo. @ImmanuelSegol